src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -369,7 +369,13 @@ class MoreInfoLight extends LitElement {
       return;
     }
 
-    this._wvSliderValue = wv;
+    if (name === "wv") {
+      this._wvSliderValue = wv;
+    } else if (name === "cw") {
+      this._cwSliderValue = wv;
+    } else if (name === "ww") {
+      this._wwSliderValue = wv;
+    }
```

Comment:
```suggestion
	if (["wv", "cw", "ww"].includes(name)) {
      this._wwSliderValue = wv;
	}
```

Reply:
No that is not the same, you could do:

```js
this[`_${name}sliderValue`] = wv;

Reply:
great...! I read the `this._*` parts as identical :coffee: